zkevm-circuits/src/witness/tx.rs

```diff
@@ -306,6 +308,38 @@ impl Transaction {
                 Value::known(F::zero()),
                 Value::known(F::from(self.tx_type as u64)),
             ],
+            [
+                Value::known(F::from(self.id as u64)),
+                Value::known(F::from(TxContextFieldTag::AccessListAddressesLen as u64)),
+                Value::known(F::zero()),
+                Value::known(F::from(access_list_address_size)),
+            ],
+            [
+                Value::known(F::from(self.id as u64)),
+                Value::known(F::from(TxContextFieldTag::AccessListStorageKeysLen as u64)),
+                Value::known(F::zero()),
+                Value::known(F::from(access_list_storage_key_size)),
+            ],
+            [
+                Value::known(F::from(self.id as u64)),
+                Value::known(F::from(TxContextFieldTag::AccessListGasCost as u64)),
+                Value::known(F::zero()),
+                rlc_be_bytes(&self.rlp_signed, challenges.keccak_input()),
```

Comment:
Is this correct?

Reply:
Sorry, it's wrong copy. Just remove this with `AccessListGasCost` in commit [6880378](https://github.com/scroll-tech/zkevm-circuits/pull/1045/commits/68803783b1447b8fefe2c1fabd626beae51b8a83).